### PR TITLE
Ethscriptions Example Added

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: Ethereum/ethscriptions/project.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  dataSources: [
+    {
+      kind: EthereumDatasourceKind.Runtime,
+      startBlock: 18130011,
+      mapping: {
+        file: "./dist/index.js",
+        handlers: [
+          {
+            kind: EthereumHandlerKind.Call,
+            handler: "handleTransaction",
```
Comment (stwiname) [inline Ethereum/ethscriptions/project.ts:51]: I haven't tested this but if you filter the function this should greatly speed up indexing by being able to use dictionaries. 

The function in this case is `data:` encoded in hex

```suggestion
            handler: "handleTransaction",
            filter: {
              function: "0x64617461"
            }
```

File: Ethereum/ethscriptions/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+You can observe the three services start, and once all are running (it may take a few minutes on your first start), please open your browser and head to [http://localhost:3000](http://localhost:3000) - you should see a GraphQL playground showing with the schemas ready to query. [Read the docs for more information](https://academy.subquery.network/run_publish/run.html) or [explore the possible service configuration for running SubQuery](https://academy.subquery.network/run_publish/references.html).
+
+## Query your project
+
+For this project, you can try to query with the following GraphQL code to get a taste of how it works.
+
+```graphql
+{
+  query {
+    inscriptions {
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/README.md:44]:
```suggestion
    inscriptions (first: 10 orderBy: BLOCK_DESC) 
```

File: Ethereum/ethscriptions/project.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+     * When developing your project we suggest getting a private API key
+     * If you use a rate limited endpoint, adjust the --batch-size and --workers parameters
+     * These settings can be found in your docker-compose.yaml, they will slow indexing but prevent your project being rate limited
+     */
+    endpoint: ["https://rpc.ankr.com/eth"],
+  },
+  dataSources: [
+    {
+      kind: EthereumDatasourceKind.Runtime,
+      startBlock: 18130011,
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/project.ts:45]: Why was this start block picked? Comment on any significance?

File: Ethereum/ethscriptions/project.ts (hunk 1 above)
Comment (jamesbayly) [reply Ethereum/ethscriptions/project.ts:51]: Please test and include if working

File: Ethereum/ethscriptions/schema.graphql [hunk 4]
```diff
@@ -0,0 +1,6 @@
+type Inscription @entity {
+  id: ID!
+  block: Int!
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/schema.graphql:3]:
```suggestion
  block: Int!
```

File: Ethereum/ethscriptions/schema.graphql [hunk 5]
```diff
@@ -0,0 +1,6 @@
+type Inscription @entity {
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/schema.graphql:1]: Can we add a `created: Date!` to this

File: Ethereum/ethscriptions/src/mappings/mappingHandlers.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    let inscription: Inscription;
+    const decodedData = hexToUTF8(tx.input);
+    if (isValidDataUri(decodedData)) {
+      inscription = Inscription.create({
+        id: tx.hash,
+        data: decodedData,
+        block: tx.blockNumber,
+        creator: tx.from,
+      });
+      inscription.save();
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/src/mappings/mappingHandlers.ts:21]:
```suggestion
      await inscription.save();
```

File: Ethereum/ethscriptions/src/mappings/mappingHandlers.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import { EthereumTransaction } from "@subql/types-ethereum";
+
+export async function handleTransaction(
+  tx: EthereumTransaction
+): Promise<void> {
+  if (tx.to) {
+    if (!tx.input) {
+      return;
+    }
+    let inscription: Inscription;
```
Comment (jamesbayly) [inline Ethereum/ethscriptions/src/mappings/mappingHandlers.ts:12]: any reason why this is initialised outside of the below IF?

**Pull request merged**